boards/common/particle-mesh/Makefile.features

```diff
@@ -2,6 +2,7 @@ CPU_MODEL = nrf52840xxaa
 
 # Put defined MCU peripherals here (in alphabetical order)
 FEATURES_PROVIDED += periph_i2c
+FEATURES_PROVIDED += periph_pwm
```

Comment:
You'll also need to update Kconfig

Reply:
Dang. I even opened the PR up side by side with #15115 and still missed that. Squashed and pushed.